repository/c.json

```diff
@@ -3914,6 +3914,18 @@
 				}
 			]
 		},
+		{
+			"name": "CPParser",
+			"details": "https://github.com/hritikchaudhary/CPParser",
+			"author": "hritikchaudhary",
```

Comment:
Since this is just your GitHub username you shouldn't specify it here, package control will find it automatically.

Reply:
> Since this is just your GitHub username you shouldn't specify it here, package control will find it automatically.

Removed